CorsixTH/Lua/world.lua

```diff
@@ -578,7 +579,7 @@ function World:tickEarthquake()
     -- start of warning quake
     if self.next_earthquake.warning_timer == earthquake_warning_period then
       self.ui:beginShakeScreen(0.2)
-      self.ui:playAnnouncement(announcements[math.random(1, #announcements)])
+      self.ui:playAnnouncement(announcements[math.random(1, #announcements)], AnnouncementPriority.Critical)
```

Comment:
Example call. We need to set priority to playAnnouncement calls. If its not specified, priority 'normal' is used.

Reply:
Looks like an excellent use case.